Sources/ApodiniDeployBuildSupport/CollectedEndpointInfo.swift

```diff
@@ -18,7 +18,7 @@ public enum WellKnownEnvironmentVariables {
     /// This environment variable is only set of the web service is running as part of a managed deployment.
     public static let currentNodeId = "AD_CURRENT_NODE_ID"
     
-    /// Key for an environment variable specifying the execution mode of ApodiniDeploy, ether dump the WebService's model structur or launch the WebService with custom config
+    /// Key for an environment variable specifying the execution mode of ApodiniDeploy, ether dump the WebService's model structure or launch the WebService with custom config
     public static let executionMode = "AD_EXECUTION_MODE"
     
```

Comment:
I just noticed that i have missed this in the refactoring of deployment structure. All the static properties that serve as keys/ values for deployment related environment variables shouldn't be needed anymore and can be removed I think. But not sure if this has to be done with this PR.

Reply:
Just removing those two properties? I think I can do that 👍

Reply:
Just checked, both of those properties are used within the `AWSIntegration.swift` file(?)

Reply:
Ah, I see. I guess, i mixed it up with [WellKnownEnvironmentVariables](https://github.com/Apodini/Apodini/blob/a990489b5b392ad34d66a18da14ed3a2335d6c4c/Sources/ApodiniDeployBuildSupport/CollectedEndpointInfo.swift). 

Sure, never mind. I'll just clean them up in a later pr. 👍